src/main/java/org/checkerframework/specimin/SpeciminStateVisitor.java

```diff
@@ -62,7 +90,11 @@ public SpeciminStateVisitor(
       Set<String> usedMembers,
       Set<String> usedTypeElements,
       Map<String, Path> existingClassesToFilePath) {
-    this.targetMethods = targetMethods;
+    this.targetMethods = new HashSet<>();
+    for (String methodSignature : targetMethods) {
+      // remove spaces
+      this.targetMethods.add(methodSignature.replaceAll("\\s", ""));
```

Comment:
The AnnotationParameterTypesVisitor compares method signatures _with_ the spaces, so this could be causing some issues regarding that